go/client/simplefs_windows.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build windows
+
+package client
+
+import (
+	"context"
+	"path/filepath"
+	"strings"
+
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func doSimpleFSRemoteGlob(g *libkb.GlobalContext, ctx context.Context, cli SimpleFSInterface, path keybase1.Path) ([]keybase1.Path, error) {
+
+	var returnPaths []keybase1.Path
+	directory := filepath.ToSlash(filepath.Dir(path.Kbfs()))
+	base := filepath.Base(path.Kbfs())
+	context.TODO()
+
+	// We know the filename has wildcards at this point.
+	// kbfs list only works on directories, so build a glob from a list result.
+
+	g.Log.Debug("doSimpleFSRemoteGlob %s", path.Kbfs())
+
+	opid, err := cli.SimpleFSMakeOpid(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer cli.SimpleFSClose(ctx, opid)
+
+	err = cli.SimpleFSList(ctx, keybase1.SimpleFSListArg{
+		OpID: opid,
+		Path: keybase1.NewPathWithKbfs(directory),
```

Comment:
This function is treating stars differently based on where they occur. If we have the glob `foo/*/*.txt`, right now we're treating the second star as a wildcard, but the first star as a literal directory name.

If we support wildcards in the middle of paths, we might be signing up for operations that are very expensive, where we have to list a ton of directories and then recurse on their results, which potential network roundtrips every step of the way. It might be that we don't want to support them, or that we want to put strict limits on how much expansion we're willing to do? But either way I don't think it's correct to interpret them as literals.